modules/installation-osp-kuryr-api-scaling.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-api-scaling_{context}"]
+= Scaling clusters that use Kuryr by using Octavia
+
+If your cluster uses Kuryr, associate your cluster's API floating IP address
```

Comment:
To avoid the apostrophe (ISG), maybe:

If your cluster uses Kuryr, associate the API floating IP address  of your cluster with the pre-existing Octavia load balancer.